config/routes/api_routes.rb

```diff
@@ -8,6 +8,11 @@ def self.extended(router)
         resources :providers, only: :index
         resources :users, only: :index
       end
+
+      namespace :api, path: "api/:api_version", api_version: /v[.0-9]+/ do
+        resource :info, only: :show, controller: "info", constraints: ValidRegisterApiRoute
+        match "*url" => "base#not_found", via: :all
```

Comment:
This was added to give us control over the error handling when receiving an invalid route (otherwise Rails returns an HTML 404 response).